repository/src/main/resources/db/migration/V20211217__RDM-13008_add_column_to_role_to_access_profiles.sql

```diff
@@ -0,0 +1,4 @@
+-- add column case_access_categories
```

Comment:
Can we rename this file to match the latest standard please - `V20211217_13008__RDM-13008_add_column_to_role_to_access_profiles.sql`

Reply:
Actioned in e2aa9add04f3f367beedea20683c03decc7a5716